models/intel/text-spotting-0002-detector/description/text-spotting-0002-detector.md

```diff
@@ -34,19 +34,19 @@ This model is a Mask-RCNN-based text detector with ResNet50 backbone and additio
 1.	Name: `im_data` , shape: [1x3x768x1280]. An input image in the [1xCxHxW] format.
     The expected channel order is BGR.
 2.	Name: `im_info`, shape: [1x3]. Image information: processed image height,
-    processed image width and processed image scale with respect to the original image resolution.
+    processed image width, and processed image scale with respect to the original image resolution.
 
 ## Outputs
 
 1.	Name: `classes`, shape: [100]. Contiguous integer class ID for every
     detected object, `0` for background (no object detected).
-1.	Name: `scores`, shape: [100]. Detection confidence scores in the [0, 1] range
+2.	Name: `scores`, shape: [100]. Detection confidence scores in the [0, 1] range
```

Comment:
You don't need enumerate list, since it is done by markdown processing.

Reply:
I know, but I was told to do so for clarity.